### PR TITLE
bpo-36211: Change the output with the full URL when executing `make -C Doc/ serve`

### PR DESCRIPTION
**Pull request by matrixise**
https://bugs.python.org/issue36211

### CODE REVIEW

File: Tools/scripts/serve.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def app(environ, respond):
     path = sys.argv[1]
     port = int(sys.argv[2]) if len(sys.argv) > 2 else 8000
     httpd = simple_server.make_server('', port, app)
-    print("Serving {} on port {}, control-C to stop".format(path, port))
+    print("Serving {0} on 0.0.0.0 port {1} (http://0.0.0.0:{1}/), control-C to stop".format(path, port))
```
Comment (remilapeyre) [inline Tools/scripts/serve.py]: What do you think about using an f-string here?

Comment (matrixise) [reply Tools/scripts/serve.py]: I could use a f-string but you can not build the documentation with python 3.5 (not compliant with the f-string)

Comment (remilapeyre) [reply Tools/scripts/serve.py]: I see, thanks!

**Review by remilapeyre — APPROVED** [review]

File: Tools/scripts/serve.py (hunk 1 above)
Comment (JulienPalard) [inline Tools/scripts/serve.py]: I don't think (I may have miseed something) querying to `0.0.0.0` is well specified (while binding to to `0.0.0.0` is), I'd prefer to see 127.0.0.1 in the link (while keeping "Serving on 0.0.0.0" to indicate it's bound to all IP addresses).
```suggestion
    print("Serving {0} on 0.0.0.0 port {1} (http://127.0.0.1:{1}/), control-C to stop".format(path, port))
```

Comment (matrixise) [reply Tools/scripts/serve.py]: ok but I should to limit to the loopback interface

Comment (JulienPalard) [reply Tools/scripts/serve.py]: Why limiting to listening to localhost? You're changing the behavior here.

**Comment by JulienPalard** [comment]
While we're at it, and sorry for the ping-pongs in this issue, why the Makefile is not simply using your `-d` option of the `http.server` module?

**Comment by JulienPalard** [comment]
Maybe superseeded by https://github.com/python/cpython/pull/12408

**Pull request closed without merging**